site/en/guides/border-router/external-commissioning/prepare.md

```diff
@@ -122,6 +122,28 @@ Specific instructions on building supported platforms with GNU Autotools can be
 found in each example's
 [platform folder](https://github.com/openthread/openthread/tree/master/examples/platforms).
 
+When commissioning a Joiner, it's important to understand the following terms
+and concepts:
+
+*   **Joining Device Credential**: You'll need to provide a Passphrase to
+    commission a device, for example `J01NU5`. This Passphrase is separate
+    from the Commissioner Credential you created when forming your Thread
+    network, and has different requirements:
+
+    *   Must be a string of all uppercase alphanumeric characters (0-9 and A-Y,
+        excluding I, O, Q, and Z for readability), with a length between 6 and
+        32 characters
+    *   Used to authenticate a device during Thread Commissioning
+    *   Used with a device's EUI64 value to generate a unique QR Code
+    *   Also referred to as Join Passphrase, Joiner Password, or PSKd
```

Comment:
These 3 bullets aren't requirements, I would keep them separate from the first bullet

Reply:
Done - let me know if this works...I can still re-work it if I need to.